src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/settings/TestGenieSettingsState.kt

```diff
@@ -4,31 +4,31 @@ package nl.tudelft.ewi.se.ciselab.testgenie.settings
  * This class is the actual data class that stores the values of the Settings entries.
  */
 data class TestGenieSettingsState(
-    var showCoverage: Boolean = false,
+        var showCoverage: Boolean = false,
 
-    var sandbox: Boolean = true,
-    var assertions: Boolean = true,
-    var seed: String = "",
-    var algorithm: ContentDigestAlgorithm = ContentDigestAlgorithm.RANDOM_SEARCH,
-    var configurationId: String = "",
-    var clientOnThread: Boolean = false,
-    var junitCheck: Boolean = false,
-    var criterionLine: Boolean = true,
-    var criterionBranch: Boolean = true,
-    var criterionException: Boolean = true,
-    var criterionWeakMutation: Boolean = true,
-    var criterionOutput: Boolean = true,
-    var criterionMethod: Boolean = true,
-    var criterionMethodNoException: Boolean = true,
-    var criterionCBranch: Boolean = true,
-    var minimize: Boolean = true
+        var sandbox: Boolean = true,
+        var assertions: Boolean = true,
+        var seed: String = "",
+        var algorithm: ContentDigestAlgorithm = ContentDigestAlgorithm.RANDOM_SEARCH,
+        var configurationId: String = "",
+        var clientOnThread: Boolean = false,
+        var junitCheck: Boolean = false,
+        var criterionLine: Boolean = true,
+        var criterionBranch: Boolean = true,
+        var criterionException: Boolean = true,
+        var criterionWeakMutation: Boolean = true,
+        var criterionOutput: Boolean = true,
+        var criterionMethod: Boolean = true,
+        var criterionMethodNoException: Boolean = true,
+        var criterionCBranch: Boolean = true,
+        var minimize: Boolean = true
 ) {
     private object DefaultSettingsState {
         const val sandbox: Boolean = true
         const val assertions: Boolean = true
         const val junitCheck: Boolean = false
         const val minimize: Boolean = true
-        val algorithm: ContentDigestAlgorithm = ContentDigestAlgorithm.RANDOM_SEARCH
+        val algorithm: ContentDigestAlgorithm = ContentDigestAlgorithm.DYNAMOSA
```

Comment:
Was this change requested by the client?

Reply:
Yes, I remember them saying explicitly that DYNAMOSA is the default for them.

Reply:
Yes, I believe they said `dynamosa` is the default (or at least the one that is always used). Technically, they also briefly said `random` is fine, but sticking to the "industry standard" sounded more reasonable